src/containers/Review/assignment/AssignmentSectionRow.tsx

```diff
@@ -138,7 +136,7 @@ const AssignmentSectionRow: React.FC<AssignmentSectionRowProps> = ({
               setIsUnassignment={() => showModal({ onConfirm: () => unassignAssignee() })}
             />
           ) : (
-            assignmentOptions.options.length > 1 && (
+            assignmentOptions.options.length > 0 && (
```

Comment:
Not sure I understand this change. Shouldn't it still show if there's only 1 option?

Reply:
Yes, so it would show if there is at least one option (instead of at least 2 options as before - because it was considering the NOT ASSIGNED not selectable option)

Reply:
Ah right -- got it. 👍